app/controllers/api/v0/articles_controller.rb

```diff
@@ -1,9 +1,9 @@
 module Api
   module V0
     class ArticlesController < ApiController
-      before_action :authenticate_with_api_key_or_current_user!, only: %i[create update]
-      before_action :authenticate!, only: :me
+      before_action :authenticate!, only: %i[create update me]
       before_action -> { doorkeeper_authorize! :public }, only: %w[index show], if: -> { doorkeeper_token }
+      before_action -> { doorkeeper_authorize! :write_articles }, only: %w[create update], if: -> { doorkeeper_token }
```

Comment:
Will tokens that have previously been created have to "update" themselves somehow to get this new permission?

Reply:
Create and update previously can only be accessed via cookie and API token. This is new permission and will not be breaking existing access.